drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -562,6 +571,7 @@ int stm32_clock_control_init(const struct device *dev)
 		}
 	}
 
+
```

Comment:
Do we need this blank line?

Reply:
No indeed. Anyway, I need to review this commit